packages/mobile/src/tokens/selectors.ts

```diff
@@ -19,7 +19,7 @@ export const tokensByAddressSelector = createSelector(
       tokenBalances[tokenAddress] = {
         ...storedState,
         balance: new BigNumber(storedState.balance),
-        usdPrice: usdPrice.isNaN() ? new BigNumber(0) : usdPrice,
+        usdPrice: usdPrice.isNaN() ? null : usdPrice,
```

Comment:
I'm concern about showing prices that were fetched long time ago. Right now, if we can't calculate the price for a token we are not deleting its previous usdPrice. I see two options here, one would be to remove every usdPrice before calculating new prices in the cloud function code. Other option would be to use the `priceFetchedAt` property to see if the price is too old.
In any case, we could do that in another PR, but I just wanted you to remind you that's something we need to address.

Reply:
I can address it here, just add a condition based on `priceFetchedAt` and set the `usdPrice` as `null` if it's too old (is 1 day too much?)